package.json

```diff
@@ -242,7 +239,7 @@
   "scripts": {
     "check": "eslint --ext .js,.jsx,.tsx,.ts . --quiet --cache",
     "fix": "eslint --ext .js,.jsx,.tsx,.ts . --quiet --fix --cache",
-    "build": "cross-env NODE_ENV=production webpack --display-error-details --verbose",
```

Comment:
These extra attributes aren't supported any more. They were added for Jenkins for some reason, so hopefully they're not needed for CircleCI or GitLab